### PR TITLE
Clean up tutorial

### PR DESCRIPTION
**Pull request by alok**
This mostly just makes the formatting a bit easier to skim.

### CODE REVIEW

File: TUTORIAL.md [hunk 1]
```diff
@@ -2,62 +2,73 @@
 
 `cargo install voidmap`
 
-if you don't have cargo, an easy way to get and manage
-it is [via rustup](https://www.rustup.rs/). Ensure that `~/.cargo/bin`
-is in your `$PATH` afterward, so that you can use the `rustup` and `cargo`
+if you don't have cargo, an easy way to get and manage it is [via
+rustup](https://www.rustup.rs/). Ensure that `~/.cargo/bin` is in your
+`$PATH` afterward, so that you can use the `rustup` and `cargo`
```
Comment (igalic) [inline TUTORIAL.md]: if you are going to add linebreaks, i suggest breaking at the end of a sentence.
that makes future editing easier on the diffs

File: TUTORIAL.md [hunk 2]
```diff
@@ -2,62 +2,73 @@
...
 
-
 #### basic node creation
-1. C-n # create new node (C-x means Ctrl+x)
-1. type some text to populate it (empty nodes are automatically deleted when unselected)
-1. Tab # create subnode
-1. type some text to populate it
-1. Enter # create sibling node
-1. type text to populate it
+
```
Comment (igalic) [inline TUTORIAL.md]: Good thinking to add a linebreak here
that makes this markdown render in other engines as well

File: TUTORIAL.md [hunk 3]
```diff
@@ -2,62 +2,73 @@
...
+4.  `Enter`: create a sibling node
 
 #### navigation
-1. C-n # create another new node and add text
-1. left/right arrow # jump between nodes
-1. C-u # search by typing text and hitting enter. repeat to show next result.
-1. C-f # search for visible nodes by entering its first letter
-1. <letter of match> # letters will show up next to options
+
+1.  `C-n`: create another new node and add text
```
Comment (igalic) [inline TUTORIAL.md]: this solves a mystery. i was wondering what the `#` was supposed to do

File: TUTORIAL.md (hunk 1 above)
Comment (alok) [reply TUTORIAL.md]: I used pandoc as an autoformatter, and it sets hard linebreaks by default. Soft may well be the better idea here.

**Comment by spacejam** [comment]
Thanks for clarifying the tutorial :)

**Pull request merged**